tracer/test/Datadog.Trace.Tests/Headers/Ip/RequestHeadersHelpersTests.cs

```diff
@@ -43,6 +43,8 @@ public class RequestHeadersHelpersTests
             new object[] { new Dictionary<string, string> { { "user-agent", "Mozilla firefox" }, { "referer", "https://example3.com/" }, { "header-custom-1", "93.12.13.14:81" }, { "x-forwarded-for", "192.168.1.2,81.202.236.243" } }, string.Empty, "81.202.236.243", 80, "80.19.14.16", 32 },
```

Comment:
I assume that IpExtractor.RealIpFromValue can actually discard IP 192.168.1.2, right?

Reply:
Yes, we ignore private IPs would be dangerous to block them.